doc/develop/test/twister.rst

```diff
@@ -804,6 +806,119 @@ line break instead of white spaces.
 
 Most everyday users will run with no arguments.
 
+.. _twister_dt_filter_expressions:
+
+DeviceTree Filtering Expressions
```

Comment:
s/DeviceTree/Devicetree/g please, this capitalization is incorrect

Reply:
So there are now occurrences of lowercase "devicetree", which is not correct either :)